README.md

```diff
@@ -1,86 +1,156 @@
+  </a>
+</p>
+
+contentful.java - Contentful Java Delivery SDK
+==============================================
 [![Build Status](https://travis-ci.org/contentful/contentful.java.svg)](https://travis-ci.org/contentful/contentful.java/builds#)
 [![codecov](https://codecov.io/gh/contentful/contentful.java/branch/master/graph/badge.svg)](https://codecov.io/gh/contentful/contentful.java)
 
-Java SDK for [Contentful's][1] Content Delivery API.
+> Java SDK for [Content Delivery API](https://www.contentful.com/developers/docs/references/content-delivery-api/) and [Content Preview API](https://www.contentful.com/developers/docs/references/content-preview-api/). It helps you to easily access your content stored in Contentful with your Java applications.
+
+
+What is Contentful?
+-------------------
+[Contentful](https://www.contentful.com) provides a content infrastructure for digital teams to power content in websites, apps, and devices. Unlike a CMS, Contentful was built to integrate with the modern software stack. It offers a central hub for structured content, powerful management and delivery APIs, and a customizable web app that enable developers and content creators to ship digital products faster.
+
+<details open>
+<summary>Table of contents</summary>
 
-[Contentful][1] provides a content infrastructure for digital teams to power content in websites, apps, and devices. Unlike a CMS, Contentful was built to integrate with the modern software stack. It offers a central hub for structured content, powerful management and delivery APIs, and a customizable web app that enable developers and content creators to ship digital products faster.
+<!-- TOC -->
+
+- [Setup](#setup)
+  - [Snapshots](#snapshots)
+  - [Proguard](#proguard)
+- [Usage](#usage)
+  - [Fetching Resources](#fetching-resources)
+  - [Calls in Parralel](#calls-in-parallel)
+  - [Paging](#paging)
+  - [Preview Mode](#preview)
+  - [Sync](#sync)
+- [Documentation](#documentation)
+- [License](#license)
+
+</details>
+
+<!-- /TOC -->
 
 Setup
 =====
 
-Grab via Maven:
+Install the Contentful dependency:
+
+* _Maven_
 ```xml
 <dependency>
   <groupId>com.contentful.java</groupId>
   <artifactId>java-sdk</artifactId>
   <version>10.0.0</version>
 </dependency>
 ```
-or Gradle:
+
+* _Gradle_
+
 ```groovy
 compile 'com.contentful.java:java-sdk:10.0.0'
 ```
 
-The SDK requires at minimum Java 6 or Android 2.3.
+This SDK needs at least Java 7 or Android 5.
```

Comment:
|Before|After|
|------|-------|
|This SDK needs at least Java 7 or Android 5| This SDK requires Java 7 (or higher version) or Android 5.|